cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -54,6 +55,10 @@
 from xmodule.tests import DATA_DIR
 from xmodule.x_module import XModuleMixin
 
+
+class GenericXBlock(XBlock):
+    """XBlock for testing indexibility"""
+
```

Comment:
nit: Since you're creating a new XBlock for the purpose, call it `NonIndexableXBlock` just for clarity.

Reply:
:+1: